test/capture-stacktrace-duration.js

```diff
@@ -0,0 +1,67 @@
+'use strict'
```

Comment:
nit: would be nice if the test file name was based on "spanFramesMinDuration" and/or a comment at the top said this is about testing that config var.

Reply:
ah, good call.  Also, dings my "I really ought to check the specs for env names _before_ picking one" bell 🛎️